runtime-modules/forum/src/lib.rs

```diff
@@ -577,7 +646,24 @@ decl_module! {
         }
 
         /// Create new thread in category with poll
-        #[weight = 10_000_000] // TODO: adjust weight
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (W + V + X + Y)` where:
+        /// - `W` is the category depth
+        /// - `V` is the length of the thread title.
+        /// - `X` is the length of the thread text.
+        /// - `Y` is the number of poll alternatives.
+        /// - DB:
+        ///    - O(W)
+        /// # </weight>
+        #[weight = WeightInfoForum::<T>::create_thread(
```

Comment:
This weight function is hard to read. Do you mind creating a separate function for this or format it in-place?

Reply:
Done https://github.com/Joystream/joystream/pull/1856/commits/dccbf2fb47f4caf123e3a1a35c9dbba01b5c88dc